src/components/parameters.js

```diff
@@ -319,6 +348,27 @@ const Parameters = ({ showParameters, hideParameters, user }) => {
         );
     }
 
+    function MakeAdvancedParameterButton(prop, label, callback) {
```

Comment:
Maybe rename 'prop' parameter name to 'showAdvancedParameter' ?

Reply:
Ok. Renamed it to "showOpenIcon"